backend/geneaprove/importers/gedcomimport.py

```diff
@@ -159,16 +159,7 @@ def _process_HEAD(self, head, filename):
                 self.ignored(f, prefix='HEAD')
 
         date_str = date.strftime('%Y-%m-%d %H:%M:%S %Z') if date else None
-        title = '"%s", %s, exported from %s, created on %s, imported on %s' % (
-            os.path.basename(name),
-            self._researcher.name,
-            created_by,
-            date_str,
-            datetime.datetime.now(
-                django.utils.timezone.get_default_timezone()).
-                strftime('%Y-%m-%d %H:%M:%S %Z')
-        )
-
+        title = f'"{os.path.basename(name)}", {self._researcher.name}, exported from {created_by}, created on {date_str}, imported on {datetime.datetime.now(django.utils.timezone.get_default_timezone()).strftime("%Y-%m-%d %H:%M:%S %Z")}'
```

Comment:
Can we keep short lines with f-strings ?

Reply:
That was bothering me as well, so I read up a little more. Turns out, of course we can! :)
(Though that long datetime call doesn't handle breaking up well. I moved that to the `imported_date_time` variable and include that in the f-string instead. I'll submit these changes once finished reviewing.

Side note: I want to go through these carefully, so it might be tomorrow before they are all addressed.)

```
        imported_date_time = datetime.datetime.now(
            django.utils.timezone.get_default_timezone()).strftime(
                "%Y-%m-%d %H:%M:%S %Z")

        title = (
            f'"{os.path.basename(name)}", '
            f'{self._researcher.name}, '
            f'exported from {created_by}, '
            f'created on {date_str}, '
            f'imported on {imported_date_time}')
```